.github/workflows/dailies.yml

```diff
@@ -13,22 +13,39 @@ jobs:
       AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
       S3_GIN_BUCKET: ${{ secrets.S3_GIN_BUCKET }}
+      CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
   run-daily-doc-link-checks:
     uses: ./.github/workflows/test-external-links.yml
 
-  notify:
+  notify-test-failure:
     runs-on: ubuntu-latest
-    needs: [run-daily-tests, run-daily-doc-link-checks]
-    if: ${{ always() && contains(needs.*.result, 'failure') }}
+    needs: [run-daily-tests]
+    if: ${{ needs.run-daily-tests.result == 'failure' }}
     steps:
     - uses: dawidd6/action-send-mail@v3
       with:
         server_address: smtp.gmail.com
-        server_port: 465  # TSL
+        server_port: 465
         username: ${{ secrets.MAIL_USERNAME }}
         password: ${{ secrets.MAIL_PASSWORD }}
-        subject: NeuroConv Daily Failure
-        to: ${{ secrets.DAILY_FAILURE_EMAIL_LIST }}  # add more with commas, no separation
+        subject: NeuroConv Daily Test Failure
+        to: ${{ secrets.DAILY_FAILURE_EMAIL_LIST }}
         from: NeuroConv
-        body: "The daily workflow for the NWB Inspector failed: please check status at https://github.com/catalystneuro/neuroconv/actions/workflows/dailies.yml"
+        body: "The daily test workflow failed: please check status at https://github.com/catalystneuro/neuroconv/actions/workflows/dailies.yml"
+
+  notify-link-check-failure:
+    runs-on: ubuntu-latest
+    needs: [run-daily-doc-link-checks]
+    if: ${{ needs.run-daily-doc-link-checks.result == 'failure' }}
+    steps:
+    - uses: dawidd6/action-send-mail@v3
+      with:
+        server_address: smtp.gmail.com
+        server_port: 465
+        username: ${{ secrets.MAIL_USERNAME }}
+        password: ${{ secrets.MAIL_PASSWORD }}
+        subject: NeuroConv Daily Doc Link Check Failure
+        to: ${{ secrets.DAILY_FAILURE_EMAIL_LIST }}
```

Comment:
You can add a new secret with my email there.

Reply:
Would you like to be on both or just the docs one?

Reply:
Add me on both, that's fine. But I will prioritze the documentation ones as that is what I promised. 

The other one will probably be useful for spikeinterface failures though.